tests/opentelemetry-docker-tests/tests/redis/test_redis_functional.py

```diff
@@ -115,3 +115,31 @@ def test_parent(self):
             child_span.attributes.get("service"), self.test_service
         )
         self.assertEqual(child_span.name, "GET")
+
+
+class TestRedisDBIndexInstrument(TestBase):
+    def setUp(self):
+        super().setUp()
+        self.redis_client = redis.Redis(port=6379, db=10)
+        self.redis_client.flushall()
+        RedisInstrumentor().instrument(tracer_provider=self.tracer_provider)
+
+    def tearDown(self):
+        super().tearDown()
+        RedisInstrumentor().uninstrument()
+
+    def _check_span(self, span, name):
+        self.assertEqual(span.name, name)
+        self.assertIs(span.status.status_code, trace.status.StatusCode.UNSET)
+        self.assertEqual(span.attributes["net.peer.name"], "localhost")
+        self.assertEqual(span.attributes["net.peer.ip"], 6379)
+        self.assertEqual(span.attributes["db.redis.database_index"], 10)
+        self.assertNotIn("db.name", span.attributes)
+
+    def test_get(self):
+        self.assertIsNone(self.redis_client.get("paneer"))
```

Comment:
Lol, I don't mind `paneer` at all but `foo` might convey that it's just a random string better 😝

Reply:
Prev tests had `cheese` so just changed it to another related food name here. No worries. I will change this to `foo`.